src/OmniSharp.Abstractions/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "version": "1.0.0-*",
   "buildOptions": {
+    "preserveCompilationContext": true,
```

Comment:
I'd love to chat sometime about why this is needed. It *really* balloons the published image with tons of binaries.